api/scpca_portal/models/project.py

```diff
@@ -332,8 +332,11 @@ def load_libraries(self):
             # Multiplexed samples are represented in scpca_sample_id as comma separated lists
             # This ensures that all samples with be related to the correct library
             for sample_id in library_metadata["scpca_sample_id"].split(","):
-                sample = self.samples.get(scpca_id=sample_id)
-                Library.bulk_create_from_dicts([library_metadata], sample)
+                # We create samples based on what is in samples_metadata.csv
+                # If the sample folder is in the input bucket, but not listed
+                # we should skip creating that library as the sample won't exist.
```

Comment:
```suggestion
                # It is possible that we are iterating over a library's metadata file
                # for a sample that was intentionally left out of samples_metadata.csv.
                # If a sample was not created we should skip creation of any libraries for that sample.
```

Alternate comment copy to help explain why a sample might not exist.